From ef3d5bcbd02abb4994fbc8dc2f248e9213ae2a1e Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Wed, 25 Oct 2017 13:18:20 -0700
Subject: [PATCH] DOCKER-1120 test cli-attach fails when it has to download the
 ubuntu image Reviewed by: Dave Eddy <dave.eddy@joyent.com> Approved by: Dave
 Eddy <dave.eddy@joyent.com>

---
 test/integration/cli-attach.test.js | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/test/integration/cli-attach.test.js b/test/integration/cli-attach.test.js
index 6624a86..29be986 100644
--- a/test/integration/cli-attach.test.js
+++ b/test/integration/cli-attach.test.js
@@ -57,6 +57,16 @@ test('setup', function (tt) {
             t.end();
         });
     });
+
+    tt.test('pull ubuntu image', function (t) {
+        h.ensureImage({
+            name: 'ubuntu:latest',
+            user: ALICE
+        }, function (err) {
+            t.error(err, 'should be no error pulling image');
+            t.end();
+        });
+    });
 });
 
 
-- 
2.21.0

